controllers/bounce_processes.go

```diff
@@ -263,6 +263,10 @@ func getProcessesReadyForRestart(logger logr.Logger, cluster *fdbv1beta2.Foundat
 		expectedProcesses += counts.Storage * (cluster.Spec.StorageServersPerPod - 1)
 	}
 
+	if cluster.Spec.TLogProcessesPerPod > 1 {
```

Comment:
I'm okay with copy and paste here but in general it would be better to have a method here to reuse the same logic.